x-pack/plugins/enterprise_search/public/applications/workplace_search/components/overview/overview_logic.ts

```diff
@@ -170,12 +159,11 @@ export const OverviewLogic = kea({
     hasErrorConnecting: [
       false,
       {
-        setHasErrorConnecting: (_: IOverviewServerData, { hasErrorConnecting }: IOverviewValues) =>
-          hasErrorConnecting,
+        setHasErrorConnecting: (_, { hasErrorConnecting }) => hasErrorConnecting,
       },
     ],
   }),
-  selectors: ({ selectors }: { selectors: IOverviewValues }) => ({
+  selectors: ({ selectors }): TKeaSelectors<IOverviewValues> => ({
     hideOnboarding: [
       () => [
         selectors.hasUsers,
```

Comment:
Similarly if you try to change this to something that isn't defined in IOverviewValues:

```suggestion
        selectors.doesntExist,
```
You'll get: `Property 'doesntExist' does not exist on type 'IOverviewValues'.`